spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestNewRewriteDataFilesAction.java

```diff
@@ -1054,6 +1045,41 @@ private void writeDF(Dataset<Row> df) {
         .save(tableLocation);
   }
 
+  private List<DeleteFile> writePosDeletesToFile(Table table, DataFile dataFile,
+                                                 int outputDeleteFiles, int rowsPerDelete) {
```

Comment:
rowsPerDeleteFile is always 1? Can we remove it?

Reply:
I was thinking it might be a helpful option to have, but we can always add later. Removed.